server/service/osquery_utils/queries.go

```diff
@@ -555,14 +555,20 @@ var extraDetailQueries = map[string]DetailQuery{
 		DirectIngestFunc: directIngestChromeProfiles,
 		Discovery:        discoveryTable("google_chrome_profiles"),
 	},
-	"battery": {
+	"battery_macos": {
 		Query:            `SELECT serial_number, cycle_count, health FROM battery;`,
 		Platforms:        []string{"darwin"},
 		DirectIngestFunc: directIngestBattery,
 		// the "battery" table doesn't need a Discovery query as it is an official
 		// osquery table on darwin (https://osquery.io/schema/5.3.0#battery), it is
 		// always present.
 	},
+	"battery_windows": {
+		Query:            `SELECT serial_number, cycle_count, designed_capacity, max_capacity FROM battery`,
+		Platforms:        []string{"windows"},
+		DirectIngestFunc: directIngestBattery,
+		Discovery:        discoveryTable("battery"),
```

Comment:
Nit: Let's add a comment that `battery` was added to Windows in osquery [5.12.1](https://github.com/osquery/osquery/releases/tag/5.12.1).